netmaster/master/netmaster.go

```diff
@@ -36,6 +36,13 @@ const (
 	defaultSkyDNSImage  = "skynetservices/skydns:latest"
 )
 
+// Run Time config of netmaster
+type nmRunTimeConf struct {
+	clusterMode string
+}
+
+var masterRTCfg nmRunTimeConf
```

Comment:
This is basically environment information. I don't think we should save it in etcd. If the netmaster is restarted, that would be still in the same environment.